Flaky test: TestServiceGet

Windows https://jenkins.dockerproject.org/job/Docker-PRs-WoW-RS1/21104/console failed on a test that looks racy; seen it fail on some others, and looks like it's comparing timestamps (which can obviously fail);


```
17:47:17 ok  	github.com/docker/docker/volume/mounts	0.084s	coverage: 66.9% of statements
17:47:17 --- FAIL: TestServiceGet (0.09s)
17:47:17 	service_test.go:144: assertion failed: 
17:47:17 		{*types.Volume}.CreatedAt:
17:47:17 			-: "2018-06-08T17:46:53Z"
17:47:17 			+: "2018-06-08T17:46:54Z"
17:47:17 		
17:47:17 time="2018-06-08T17:46:54Z" level=warning msg="could not determine size of volume" error="GetFileAttributesEx fake: The system cannot find the file specified." volume=test
17:47:17 time="2018-06-08T17:46:54Z" level=warning msg="could not determine size of volume" error="GetFileAttributesEx fake: The system cannot find the file specified." volume=test
17:47:17 time="2018-06-08T17:46:54Z" level=warning msg="could not determine size of volume" error="GetFileAttributesEx fake: The system cannot find the file specified." volume=test
17:47:17 time="2018-06-08T17:46:54Z" level=warning msg="could not determine size of volume" error="GetFileAttributesEx fake: The system cannot find the file specified." volume=test3
17:47:17 time="2018-06-08T17:46:54Z" level=warning msg="could not determine size of volume" error="GetFileAttributesEx fake: The system cannot find the file specified." volume=test
17:47:17 FAIL
```

--- a/volume/testutils/testutils.go
+++ b/volume/testutils/testutils.go
@@ -41,11 +41,12 @@ func (NoopVolume) CreatedAt() (time.Time, error) { return time.Now(), nil }
 type FakeVolume struct {
 	name       string
 	driverName string
+	createdAt  time.Time
 }
 
 // NewFakeVolume creates a new fake volume for testing
 func NewFakeVolume(name string, driverName string) volume.Volume {
-	return FakeVolume{name: name, driverName: driverName}
+	return FakeVolume{name: name, driverName: driverName, createdAt: time.Now()}
 }
 
 // Name is the name of the volume
@@ -69,7 +70,9 @@ func (FakeVolume) Status() map[string]interface{} {
 }
 
 // CreatedAt provides the time the volume (directory) was created at
-func (FakeVolume) CreatedAt() (time.Time, error) { return time.Now(), nil }
+func (f FakeVolume) CreatedAt() (time.Time, error) {
+	return f.createdAt, nil
+}
 
 // FakeDriver is a driver that generates fake volumes
 type FakeDriver struct {

